install/k8s/configs/etcd.yaml

```diff
@@ -0,0 +1,99 @@
+---
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: contiv-etcd
+  namespace: kube-system
+  labels:
+    k8s-app: contiv-etcd
+spec:
+  selector:
+    matchLabels:
+      k8s-app: contiv-etcd
+  template:
+    metadata:
+      labels:
+        k8s-app: contiv-etcd
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      hostPID: true
+      nodeSelector:
+        node-role.kubernetes.io/master: ''
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+      initContainers:
+        - name: contiv-etcd-init
+          image: ferest/etcd-initer:latest
```

Comment:
this should be pinned or even rebuilt and in our hub?

Reply:
what was wrong with just using args for the etcd container?

Reply:
explained at README of https://hub.docker.com/r/ferest/etcd-initer/

Reply:
this is currently not pined ( i guess won't be updated anyway like the sidecar project), I didn't make it in contiv dockerhub was because by that time i wasn't able to link it with contiv dockerhub. we can revisit it